dds/DdsDcpsCore.idl

```diff
@@ -77,28 +77,28 @@ module DDS {
     };
     typedef sequence<BinaryProperty_t> BinaryPropertySeq;
 
-    @appendable
+    //@appendable
```

Comment:
Add a comment for why this is here.

I think these need to be XCDR1 if/when we support that (Annex D IDL, comment at the top).